debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -278,6 +278,8 @@ private ValueConverter createArrayConverter(Column column, Field fieldDefn) {
                 .jdbcType(schema.columnTypeNameToJdbcTypeId(PostgresSchema.parse(elementTypeName).table()))
                 .type(elementTypeName)
                 .optional(true)
+                .scale(column.scale())
```

Comment:
Should we check for the value to be not -1?

Reply:
`-1` is a valid value for varibale scaled decimal, so this is intentional

Reply:
Ok, seems legit. Btw. in the constructor of `ColumnImpl` there're two asserts rejecting anything < -1. That's potentially not correct then if it's a variable scale decimal column?